client/src/components/AlternateApp.jsx

```diff
@@ -215,6 +216,8 @@ export default class AlternateApp extends React.Component {
                 )}
               />
 
+              <SecureRoute path="/toyproblems" render={() => <ToyProblems />} />
```

Comment:
What is the difference for SecureRoute?

Reply:
SecureRoute requires authentication to render the component.
https://www.npmjs.com/package/@okta/okta-react